dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/ProcessInstanceMapper.xml

```diff
@@ -168,14 +168,14 @@
         select *
         from t_ds_process_instance
         where 1=1
```

Comment:
Hi, is it possible to delete this statement?

Reply:
thank you for reminding!
"1=1" is not necessary.